SAMtags.tex

```diff
@@ -170,7 +172,7 @@ \subsection{Additional Template and Mapping data}
 in SAM.
 
 \item[IH:i:\tagvalue{count}]
-Number of stored alignments in SAM that contains the query in the current
+Number of alignments stored in the file that contain the query in the current
```

Comment:
can you expand here on the definition of query. given the term `queryname` I am not 100% clear about whether this pertains to only one "read", or to all the reads from the template.

Reply:
I would guess that “query in the current record” implies it's just this read's sequence, and there's samtools-devel emails from 2011 that can be interpreted as agreeing with that. BWA doesn't implement NH/IH/HI, and HISAT2 seems to calculate NH in both interpretations in different circumstances. So I'd rather not expand on this without doing further research!

Reply:
I haven't any objections to the change, but in review I did wonder how this tag is ever meant to work...

"Stored" is a dangerous word.  If we do a sub-range query then we likely invalidate this tag, and not doing so turns sub-range queries from fast operations into complex nightmares.  It's simply not a sensible tag as it's written and doesn't match common practice.

I'd guess than "Number of generated alignment records" may match the expected usage more where the aligner can inform us of how many records it wrote, but as you say its usage right now is patchy and probably ill defined.  The language clarification is fine and I don't mind punting more substantive changes to another PR.